training/trillium/Llama3-70B-PyTorch/GCE/README.md

```diff
@@ -34,6 +34,8 @@ and comes with all the package dependency needed to run the model training.
 All the command below should run from your own machine (not the TPU host you
 created).
 
+Note: if want to get metric printed on all workers, please try `us-central1-docker.pkg.dev/deeplearning-images/reproducibility/pytorch-xla/llama3-70b:jan15built` by modifying [this line](https://github.com/AI-Hypercomputer/tpu-recipes/blob/af4393a42966a9a1f76b9f270a2e07788e2879ff/training/trillium/Llama3-70B-PyTorch/GCE/host.sh#L3)
```

Comment:
I think you can simply replace the older docker URL a few lines above if you tested that the perf is the same. Alternatively, maybe you could move this to a "# Appendix: historical docker image releases" section.

Reply:
thanks, updated